example/passthrough_hp.cc

```diff
@@ -772,7 +774,7 @@ static void do_readdir(fuse_req_t req, fuse_ino_t ino, size_t size,
         if (entsize > rem) {
             if (fs.debug)
                 cerr << "DEBUG: readdir(): buffer full, returning data. " << endl;
-            if (plus)
+            if (did_lookup)
```

Comment:
Thanks for fixing this. An alternative check could be `if (e.ino)` instead of needing the did_lookup variable.